hist/hist/src/HFitImpl.cxx

```diff
@@ -1016,20 +1023,25 @@ TFitResultPtr ROOT::Fit::FitObject(THnBase * s1, TF1 *f1 , Foption_t & foption ,
 
 // function to compute the simple chi2 for graphs and histograms
 
-double ROOT::Fit::Chisquare(const TH1 & h1,  TF1 & f1, bool useRange, bool usePL) {
-   return HFit::ComputeChi2(h1,f1,useRange, usePL);
+
+double ROOT::Fit::Chisquare(const TH1 & h1,  TF1 & f1, bool useRange, ROOT::Fit::EChisquareType type) {
+   return HFit::ComputeChi2(h1,f1,useRange, type);
 }
 
 double ROOT::Fit::Chisquare(const TGraph & g, TF1 & f1, bool useRange) {
-   return HFit::ComputeChi2(g,f1, useRange, false);
+   return HFit::ComputeChi2(g,f1, useRange, kNeyman);
 }
 
 template<class FitObject>
-double HFit::ComputeChi2(const FitObject & obj,  TF1  & f1, bool useRange, bool usePL ) {
+double HFit::ComputeChi2(const FitObject & obj,  TF1  & f1, bool useRange, ROOT::Fit::EChisquareType type ) {
 
    // implement using the fitting classes
    ROOT::Fit::DataOptions opt;
-   if (usePL) opt.fUseEmpty=true;
+   if (type != ROOT::Fit::kNeyman) opt.fUseEmpty=true;  // use empty bin when not using Neyman chisquare (observed error)
```

Comment:
```suggestion
   opt.fUseEmpty = type != ROOT::Fit::kNeyman;  // use empty bin when not using Neyman chisquare (observed error)
```
I think that would make the code more readable, because like you have it now it's not clear what the initial value of `opt.fUseEmpty` is. But if you always set it explicitly, this is clearer.